### DISCUSSION TITLE
move each exporter's scripts (photoshop, gimp, krita, godot) into a directory (such as "/plugins" or "/scripts")

### DISCUSSION THREAD

**Discussion by Aodaruma** [Aodaruma/coa_tools2#3] [Ideas]
I'm considering whether it would be better to organize them together in another plugin folder because it doesn't look very clean to have them in the root directory.
If we adopt Python's flat layout, it might be better to manage them with that approach. 
https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/

Alternatively, should I create a separate repository for these?

**Comment by tozpeak** [comment]
Separate repositories for importer plugins could be unnecessary complexity and atomization. If any breaking changes are introduced into the tool, it would be easier to implement those changes in one place, or at least create issues for them to track the needed work.

However, I support that they probably should be relocated from root for a better order.

Also, list of engines supported to import might grow, so maybe Godot folder should be moved to /engines (or something) folder from root as well.

**Comment by UPSIDEDOWNMR** [comment]
I wish to do some tutorials on this tool as a comparison to spine. Currently I am having issues when I edit mesh and after the texture disappears. Any thoughts on why this is happening?
